environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/NetworkCreationValidator.java

```diff
@@ -43,12 +41,22 @@ public ValidationResultBuilder validateNetworkEdit(Environment environment, Netw
         return resultBuilder;
     }
 
+    private void validateNetwork(Environment environment, NetworkDto network, ValidationResultBuilder resultBuilder) {
+        if (network != null) {
+            EnvironmentNetworkValidator environmentNetworkValidator =
+                    environmentNetworkValidatorsByCloudPlatform.get(CloudPlatform.valueOf(environment.getCloudPlatform().toUpperCase()));
+            if (environmentNetworkValidator != null) {
+                environmentNetworkValidator.validateDuringRequest(network, resultBuilder);
+            }
+        }
+    }
+
```

Comment:
this is the same code as the overloaded method below

Reply:
it's about to change due to gtopolyai's request